src/env.js

```diff
@@ -41,7 +41,7 @@ export const env = createEnv({
       .optional()
       // transform to boolean using preferred coercion logic
       .transform((s) => s !== undefined && s !== "false" && s !== "0"),
-    HIGHLIGHT_PROJECT_ID: z.string().default("6gl9mxzg"),
+    HIGHLIGHT_PROJECT_ID: z.string().default("jdkmlr3g"),
```

Comment:
hum what is this for?

Reply:
Ah was created a new project with "Flojoy-cloud" name as previous one has typo in name and could't change the name. I'll revert this anyway. You probably can fix that typo.